WordPressKitTests/Utilities/WordPressAPIErrorTests.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import XCTest
+@testable import WordPressKit
+
+class WordPressAPIErrorTests: XCTestCase {
+
+    func testLocalizedMessage() {
+        struct TestError: LocalizedError {
+            var errorDescription: String? = "this is a test error"
+        }
+
+        let error = WordPressAPIError.endpointError(TestError())
+        XCTAssertEqual((error as NSError).localizedDescription, "this is a test error")
```

Comment:
For consistency with the test below, what do you think of adding

```suggestion
        XCTAssertEqual(error.errorDescription, "this is a test error")
        XCTAssertEqual((error as NSError).localizedDescription, "this is a test error")
```